### PR TITLE
Added Serbian Cyrillic support with additional bug fixes

### PR DESCRIPTION
**Pull request by aleksandarni**

### CODE REVIEW

File: src/main/java/pl/allegro/finance/tradukisto/MoneyConverters.java [hunk 1]
```diff
@@ -35,8 +36,9 @@
     LATVIAN_BANKING_MONEY_VALUE(latvianContainer().getBankingMoneyConverter()),
     KAZAKH_BANKING_MONEY_VALUE(kazakhContainer().getBankingMoneyConverter()),
     UKRAINIAN_BANKING_MONEY_VALUE(ukrainianContainer().getBankingMoneyConverter()),
-    TURKISH_BANKING_MONEY_VALUE(turkishContainer().getBankingMoneyConverter()),
```
Comment (MaciejLipinski) [inline src/main/java/pl/allegro/finance/tradukisto/MoneyConverters.java:38]: revert moving Turkish converter declaration; the order is a mess, but moving just one converter doesn't help

File: src/main/java/pl/allegro/finance/tradukisto/ValueConverters.java [hunk 2]
```diff
@@ -32,7 +33,8 @@
     KAZAKH_INTEGER(kazakhContainer().getNumbersConverter()),
     UKRAINIAN_INTEGER(ukrainianContainer().getNumbersConverter()),
     SERBIAN_INTEGER(serbianContainer().getNumbersConverter()),
-    FRENCH_INTEGER(frenchContainer().getNumbersConverter()),
+    SERBIAN_CYRILLIC_INTEGER(serbianCyrillicContainer().getNumbersConverter()),
+    FRENCH_INTEGER(frenchContainer().getNumbersConverter());
```
Comment (MaciejLipinski) [inline src/main/java/pl/allegro/finance/tradukisto/ValueConverters.java:37]: change `;` to `,`

File: src/main/java/pl/allegro/finance/tradukisto/MoneyConverters.java [hunk 3]
```diff
@@ -35,8 +36,9 @@
...
     SERBIAN_BANKING_MONEY_VALUE(serbianContainer().getBankingMoneyConverter()),
+    SERBIAN_CYRILLIC_BANKING_MONEY_VALUE(serbianCyrillicContainer().getBankingMoneyConverter());
```
Comment (MaciejLipinski) [inline src/main/java/pl/allegro/finance/tradukisto/MoneyConverters.java:40]: change `;` to `,`

File: src/test/groovy/pl/allegro/finance/tradukisto/MoneyConvertersTest.groovy [hunk 4]
```diff
@@ -18,6 +18,8 @@ import static pl.allegro.finance.tradukisto.MoneyConverters.SERBIAN_BANKING_MONE
 import static pl.allegro.finance.tradukisto.MoneyConverters.SLOVAK_BANKING_MONEY_VALUE
 import static pl.allegro.finance.tradukisto.MoneyConverters.TURKISH_BANKING_MONEY_VALUE
 import static pl.allegro.finance.tradukisto.MoneyConverters.UKRAINIAN_BANKING_MONEY_VALUE
+import static pl.allegro.finance.tradukisto.MoneyConverters.SERBIAN_BANKING_MONEY_VALUE
+import static pl.allegro.finance.tradukisto.MoneyConverters.SERBIAN_CYRILLIC_BANKING_MONEY_VALUE
```
Comment (MaciejLipinski) [inline src/test/groovy/pl/allegro/finance/tradukisto/MoneyConvertersTest.groovy:22]: rearrange imports, now they're not ordered alphabetically and there's a duplicate `SERBIAN_BANKING_MONEY_VALUE` on line 17

File: src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/RegularPluralFormsTest.groovy [hunk 5]
```diff
@@ -1,6 +1,5 @@
 package pl.allegro.finance.tradukisto.internal.languages
 
-import pl.allegro.finance.tradukisto.internal.languages.GenderType
```
Comment (MaciejLipinski) [inline src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/RegularPluralFormsTest.groovy:3]: nice find!

**Review by MaciejLipinski — CHANGES_REQUESTED** [review]
Thank you for your second contribution! There's a couple of things that need fixing, please see the comments

**Comment by aleksandarni** [comment]
@MaciejLipinski Thank you for the suggestions. I believe I fixed/improved all of them.

**Review by MaciejLipinski — APPROVED** [review]

**Pull request merged**